### PR TITLE
POST request example has been added to the stateless demo app

### PR DESCRIPTION
**Pull request by feeeper**
For the #1745 issue

### CODE REVIEW

File: samples/Nancy.Demo.Authentication.Stateless/SecureModule.cs [hunk 1]
```diff
@@ -1,6 +1,7 @@
 namespace Nancy.Demo.Authentication.Stateless
 {
-    using Nancy.Demo.Authentication.Stateless.Models;
+  using System;
```
Comment (jchannon) [inline samples/Nancy.Demo.Authentication.Stateless/SecureModule.cs]: Can you indent these please

Comment (feeeper) [inline samples/Nancy.Demo.Authentication.Stateless/SecureModule.cs]: Add one more commit. Hope now is all right :)

Comment (jchannon) [inline samples/Nancy.Demo.Authentication.Stateless/SecureModule.cs]: Brill. Can you squash those 2 commits into 1. And the ill merge it :)

Thanks

**Comment by jchannon** [comment]
Looks like a rebase has gone a bit wonky :smile: 

Try a `git rebase -i master` and remove any commits that you haven't committed

**Comment by jchannon** [comment]
Thanks :heart: :smile:

**Pull request merged**